code/modules/mob/living/carbon/human/species_types/skeletons.dm

```diff
@@ -7,7 +7,7 @@
 	sexes = 0
 	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/skeleton
 	species_traits = list(NOBLOOD)
-	inherent_traits = list(TRAIT_RESISTHEAT,TRAIT_NOBREATH,TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_RADIMMUNE,TRAIT_PIERCEIMMUNE,TRAIT_NOHUNGER,TRAIT_EASYDISMEMBER,TRAIT_LIMBATTACHMENT,TRAIT_FAKEDEATH)

```

Comment:
High pressure would be okay to remove, Low pressure will need pirate modifications to ensure the first mate/gunner have pressure resistant gear. The Captain's iirc has proofing.

Reply:
Nope, as stated in the PR at the top, "THIS WILL NOT AFFECT SPACE QUEEN MEN". They get hat that are space proof already and that are almost as good as nukie gear

Reply:
Unless the body gets pressure proofing, the hat doesn't provide full body coverage.